packages/lib/har/harUtils.ts

```diff
@@ -83,23 +95,35 @@ export const toHarContent = (body: string | Buffer | null, mimeType?: string): H
   };
 };
 
-export const fromHarContent = (content?: HarFormatContent) => {
+export const fromHarContent = (saveAsString: boolean, content?: HarFormatContent) => {
```

Comment:
When reading from the har content, we should not use the setting, and instead check what it present in the har content and just use what we have.